action.js

```diff
@@ -73,27 +88,37 @@ module.exports = class {
     if (!_valid) return false;
 
     const issues = await this.getIssues();
+    console.log({ issues });
     return issues && issues.hasOwnProperty("issue");
   }
 
-  // TODO - Test this
   async getCommits() {
-    const { data } = await github.Github.listCommitsOnPullRequest({
-      repo: this.githubEvent.pull_request.repository.name,
-      pullNumber: this.githubEvent.number
-    });
+    try {
+      const { data } = await this.github.Github.pulls.listCommits({
+        owner: GITHUB_OWNER,
+        repo: this.githubEvent.pull_request.repository.name,
+        pull_number: this.githubEvent.number
+      });
 
-    return data;
+      return data;
+    } catch (e) {
+      console.error({ e });
+      return [];
```

Comment:
Should we add to to the error array here instead of sending back an empty array that will always pass the valid commits check?